client/Panels/Topics/Model.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { ChangeEvent, useState, useCallback } from 'react';
+import { GET_TOPICS } from 'Queries/Topics';
+import { useQuery } from '@apollo/client';
+import { useTopicsModelType, topicsType } from './useTopicsModel.types';
+import debounce from 'lodash.debounce';
+
+const onChangeEvent = (
+  evt: ChangeEvent<HTMLInputElement>,
```

Comment:
Don't use html events here, it breaks the MVVC abstraction. Use scalar types or add an abstraction.